test/specs/components/Component/Component-test.ts

```diff
@@ -0,0 +1,33 @@
+import { mountWithProvider as mount } from 'test/utils'
+
+import Component from 'src/components/Component/Component'
+import { isConformant } from 'test/specs/commonTests'
+
+describe('Component', () => {
+  const createComponentComp = (factoryFn: Function, val, options?) =>
+    mount(factoryFn(val, options)).find(Component)
+
+  xdescribe('is conformant', () => {
+    isConformant(Component)
+  })
+
+  describe(`create`, () => {
+    it(`renders a ${Component.defaultProps.as} element with children prop`, () => {
```

Comment:
There is a common test called `rendersChildren()` that I think we can use here.

Reply:
not in the scope of this PR as it's just renaming files; I'll try to port the `rendersChildren` tests from Semantic UI, though